src/QualificationRunner.Core/Domain/LogWatcher.cs

```diff
@@ -102,7 +96,32 @@ private bool logFileCreated()
 
          private void appendTextToLog()
          {
-            _logger.AddToLog(readFileText(), LogLevel.Debug, null);
+            var entries = readFileText()?.Split('\n');
+            entries.Each(s =>
+            {
+               var (entry, level) = logLevelFor(s);
+               _logger.AddToLog(entry, level, _logWatcherOptions.Category);
+            });
+         }
+
+         private (string, LogLevel) logLevelFor(string entry)
+         {
+            if (string.IsNullOrEmpty(entry))
+               return (entry, LogLevel.Debug);

```

Comment:
maybe `return ("", LogLevel.Debug);` would be better?

Reply:
Does not make a difference. The actual _logger.AddToLog would have received that entry otherwise and should be able to deal with it.